src/main/java/apoc/util/Util.java

```diff
@@ -700,4 +705,47 @@ public static boolean classExists(String className) {
         Object value = ApocConfiguration.get(loadType).get(key);
         return Optional.ofNullable(value).map(Object::toString);
     }
+
+    public static String dateFormat(TemporalAccessor value, String format){
+        return getFormat(format).format(value);
+    }
+
+    public static TemporalAccessor dateParse(String value, Class<? extends TemporalAccessor> date, String...formats) {
+        String methodName = "parse";
```

Comment:
Why do we have to do it like this?
Isn't that on an interface?

Reply:
Do you mean that can be invoked in this way?
```java
parseDate(value.toString(), TemporalAccessor.class, dateParse);
```

Reply:
create method-handle and cache them in a static Map<Class,MethodHandle>